askama_shared/src/generator.rs

```diff
@@ -160,13 +160,18 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         buf.writeln("Ok(())")?;
         buf.writeln("}")?;
 
-        buf.writeln("const EXTENSION: ::std::option::Option<&'static ::std::primitive::str> = ")?;
-        buf.writeln(&format!("{:?}", self.input.extension()))?;
-        buf.writeln(";")?;
-
-        buf.writeln("const SIZE_HINT: ::std::primitive::usize = ")?;
-        buf.writeln(&format!("{}", size_hint))?;
-        buf.writeln(";")?;
+        buf.writeln(&format!(
```

Comment:
Would prefer not changing style in the same commit as making functional changes.